testdata/configroot/scopes/global/descriptors.yml

```diff
@@ -100,12 +102,57 @@ metrics:
       service: 1 # STRING
       method: 1 # STRING
       response_code: 2 # INT64
+  - name: request_size
+    kind: DISTRIBUTION
+    value: INT64
+    description: request size by source, target, and service
+    buckets:
+      exponential_buckets:
+        num_finite_buckets: 8
+        scale: 1
+        growth_factor: 10
```

Comment:
Nit: Use camelCase field names here and elsewhere now that we can parse them correctly. Not required for the label names, though we should probably decide what our convention is w.r.t. label names - I lean towards keeping everything camelCased.